ci/stress_tests/stress_testing_config.yaml

```diff
@@ -91,7 +91,7 @@ setup_commands:
     # - sudo dpkg --configure -a
     # Install basics.
     - sudo apt-get update
-    - sudo apt-get install -y cmake pkg-config build-essential autoconf curl libtool unzip flex bison python
+    - sudo apt-get install -y pkg-config build-essential autoconf curl libtool unzip flex bison python
```

Comment:
Should we get rid of even more? E.g., `pkg-config`, `autoconf`, `libtool`, `flex`, `bison`, `python`

Reply:
in other places as well